src/vft.py

```diff
@@ -134,8 +142,9 @@ def generate(width, height, vft_id, tag_border_size, value, debug):
     vft_layout = VFTLayout(width, height, vft_id, tag_border_size)
     # 2. add fiduciary markers (tags) in the top-left, top-right,
     # and bottom-left corners
-    for tag_number in range(3):
+    for tag_number in range(4):
```

Comment:
can you please move this 4 to a constant?

Reply:
Good idea.